src/adapter/src/notice.rs

```diff
@@ -197,6 +200,7 @@ impl AdapterNotice {
             AdapterNotice::UnknownSessionDatabase(_) => SqlState::SUCCESSFUL_COMPLETION,
             AdapterNotice::OptimizerNotice { .. } => SqlState::SUCCESSFUL_COMPLETION,
             AdapterNotice::WebhookSourceCreated { .. } => SqlState::WARNING,
+            AdapterNotice::PerReplicaLogRead { .. } => SqlState::WARNING,
```

Comment:
What are the semantics of `WARNING` here? Thinking in log levels, this shouldn't be a warning but an info level, since this is just a heads-up message. But it doesn't look like `INFO` exists here, so maybe `WARNING` is correct.

Reply:
Ah, I see below that we set the severity to `NOTICE`, so I guess this is alright.